src/lib/utils/options/declaration.ts

```diff
@@ -246,7 +256,14 @@ export function convert<T extends DeclarationOption>(value: unknown, option: T):
         case ParameterType.String:
             return Result.Ok(value == null ? '' : String(value));
         case ParameterType.Number:
-            return Result.Ok(parseInt(String(value), 10) || 0);
+            const numberOption = option as NumberDeclarationOption;
+            const numValue = parseInt(String(value), 10) || 0;
+            if (typeof numberOption.minValue === 'number' &&  numValue < numberOption.minValue) {
+                return Result.Err(`${numberOption.name} must be >= ${numberOption.minValue}`);
```

Comment:
Is there a reason you don't include both bounds in the error message? It seems like it would be more useful:

```bash
$ typedoc --numOption 10
Error: numOption must be <= 5
$ typedoc --numOption 2
Error: numOption must be >= 3
# vs
$ typedoc --numOption 10
Error: numOption must be between 2 and 10
```

Probably best to extract the error message creation to a helper function, since there are several possible combinations.

Reply:
I agree that if both are defined it would be more useful to include both bounds in the error message.